### DISCUSSION TITLE
Is there a way for me to download all the documentation as a pdf file?

### DISCUSSION THREAD

**Discussion by victormnielsen** [gama-platform/gama#378] [Q&A]
Title says it all. Basically everything from this page: https://gama-platform.org/wiki/Home

**Comment by lesquoyb** [comment]
Yes, on the download page there's a link to download the documentation, and [here it is](https://raw.githubusercontent.com/wiki/gama-platform/gama/resources/pdf/docGAMAv193.pdf)

**Comment by RoiArthurB** [comment]
To complete the answer, you'll always find a PDF version of the documentation available on this page : https://gama-platform.org/download (the blue button on the button right of the page) :)
